python/ray/autoscaler/updater.py

```diff
@@ -298,6 +307,9 @@ def run_rsync_down(self, source, target):
             "{}@{}:{}".format(self.ssh_user, self.ssh_ip, source), target
         ])
 
+    def download_one_file(self, source, target):
```

Comment:
Since "class DockerCommandRunner(SSHCommandRunner)" we do not have to implement upload_one_file or download_one_file for DockerCommandRunner as it inherits SSHCommandRunner. It might be a bit confusing that it later calls self.run_rsync_up. For example, if you have a DockerCommandRunner that calls upload_one_file it will run SSHCommandRunner's upload_one_file, which invokes DockerCommandRunner's run_rsync_up.